develop/toolkit/blockchain/fork-chains/chopsticks/.pages

```diff
@@ -0,0 +1,4 @@
+title: Chopsticks
+nav:
+  - index.md
+  - 'Fork Live Chains with Chopsticks': fork-live-chains.md
```

Comment:
This ideally should be a Get Started page.

If we take a step back and look at the structure between Chopsticks and Zombienet, Chopsticks includes install, configuration, and examples. Whereas now in the Zombienet section, there is a page for Installation and a page for Configuration. We need to be consistent in how we document tools, which is why I proposed the structure I did in this [comment](https://github.com/polkadot-developers/polkadot-docs/pull/154#pullrequestreview-2443955760)

If I made some contradicting statement somewhere along the line, I apologize